CMakeLists.txt

```diff
@@ -57,11 +57,70 @@ endif()
 
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/cmake)
 
+# To configure mbedtls
+# todo make the configuration better
+set(MBEDTLS_CONFIG_FILE "mbedtls_config.h")
+add_compile_options(-I${CMAKE_SOURCE_DIR}/lib/include)
```

Comment:
Would it be better to use https://cmake.org/cmake/help/latest/command/include_directories.html rather than `add_compile_options` ?

Reply:
Yep - have changed it to that